lib/modules/manager/kubernetes/extract.ts

```diff
@@ -8,6 +10,10 @@ import { getDep } from '../dockerfile/extract';
 import type { ExtractConfig, PackageDependency, PackageFile } from '../types';
 import type { KubernetesConfiguration } from './types';
 
+const kubernetesApiVersions: Record<string, string[]> = JSON5.parse(
+  dataFiles.get('data/kubernetes-api.json5')!
+);
```

Comment:
we should share this, otherwise we need twice in memory. (perf)

Reply:
You mean between manager and datasource?

Reply:
yes

Reply:
manager does access datasource anyways